dev_tools/docs/sphinx/conf.py

```diff
@@ -147,5 +152,24 @@ def _order(docname):
         _rmtree_if_exists(f'{qaoa_p1_tasks.DEFAULT_BASE_DIR}/2020-03-tutorial')
 
 
+import subprocess
+
+REPO_DIR = subprocess.Popen(['git', 'rev-parse', '--show-toplevel'],
+                            stdout=subprocess.PIPE).communicate()[0].rstrip().decode('utf-8')
+
+
+def source_read(app, docname, source):
+    import re
```

Comment:
nit: why not import on top of the file?

Reply:
as I was typing it I knew someone would comment on it. The real reason is because I was doing it quickly and didn't want to scroll all the way up to the top! The plausible deniability reason is because this is a highly-customized change, I kept all the non-standard sphinx conf.py stuff into this function

Reply:
done